objectivec/GPBDictionary.h

```diff
@@ -60,55 +60,6 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic, readonly) NSUInteger count;
 
 /**
- * @return A newly instanced and empty dictionary.
- **/
-+ (instancetype)dictionary;
```

Comment:
Your call... would you prefer we left it? I am happier going with consistency (all alloc init) as opposed to having the one "weird" class method.